modules/affile/file-opener.h

```diff
@@ -47,6 +47,12 @@ typedef struct _FileOpenerOptions
   gint create_dirs;
 } FileOpenerOptions;
 
+typedef enum {
+  FILEOPENER_RESULT_SUCCESS,
```

Comment:
what do you think about an underscore between FILE and OPENER? those are separate terms, the filenames and other identifiers follow this convetion.